application/values.yaml

```diff
@@ -138,7 +167,7 @@ service:
     #   ingress.kubernetes.io/rewrite-target: /
     #   ingress.kubernetes.io/force-ssl-redirect: true
   
-  ports: []
+  ports:
   # - port: 8080
   #   name: http
   #   protocol: TCP
```

Comment:
These should be uncommented as we are using port 8080 in whole of the chart so here also this should be uncommented